specifyweb/specify/views.py

```diff
@@ -584,9 +584,9 @@ def record_merge_fx(model_name: str, old_model_ids: List[int], new_model_id: int
         # Handle case of updating a large amount of record ids in a foreign table.
         # Example: handle case of updating a large amount of agent ids in the audit logs.
         # Fix by optimizing the query by consolidating it here
-        if model_name.lower() in MERGING_OPTIMIZATION_TABLES.keys() and \
-            table_name.lower() in MERGING_OPTIMIZATION_TABLES[model_name.lower()]:
-            for field_name in MERGING_OPTIMIZATION_FIELDS[(model_name.lower(), table_name.lower())]:
+        if model_name.lower() in MERGING_OPTIMIZATION_FIELDS and \
+            table_name.lower() in MERGING_OPTIMIZATION_FIELDS[model_name.lower()][table_name.lower()]:
```

Comment:
Should be  `MERGING_OPTIMIZATION_FIELDS.keys()` and `table_name.lower() in MERGING_OPTIMIZATION_FIELDS[model_name.lower()].keys():`

Reply:
aren't they the same?

x = {'a': 90, 'b': 10}
'a' in x
-- True
 'xx' in x
-- False

Reply:
ya, tested it out, guess they are the same.  Allows thought you need '.keys()'.  Time to through out all the keys, lol.